dolphinscheduler-ui/src/views/projects/task/components/node/fields/use-worker-group.ts

```diff
@@ -43,7 +42,8 @@ export function useWorkerGroup(): IJsonItem {
     span: 12,
     name: t('project.node.worker_group'),
     props: {
-      loading: loading
+      loading: loading,
+      'on-update:value': () => (model.environmentCode = null)
```

Comment:
Setting to null, the environmentCode will not has default value after changing workgroup.